content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -18,10 +18,7 @@ see the [Creating a cluster with kubeadm](/docs/setup/production-environment/too
 This installation guide is for Kubernetes {{< skew currentVersion >}}. If you want to use a different
 Kubernetes version, please refer to the following pages instead:
```

Comment:
_this describes an ideal implementation_
I'd move this text inside the short code.

---

Here's how it could look in the Markdown
```markdown
{{< doc-versions-list item="installation guide" >}}
```

rendering to:
```html
<div class="version-list">
  <p>
      This installation guide is for Kubernetes 1.28. If you want to use a different Kubernetes version, please refer to the following pages instead:
  </p>
  <ul >
      <li>
          <a href="https://v1-27.docs.kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/">Installing kubeadm</a> (Kubernetes v1.27)
      </li>
      <li>
          <a href="https://v1-26.docs.kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/">Installing kubeadm</a> (Kubernetes v1.26)
      </li>
      <li>
          <a href="https://v1-25.docs.kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/">Installing kubeadm</a> (Kubernetes v1.25)
      </li>
      <li>
          <a href="https://v1-24.docs.kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/">Installing kubeadm</a> (Kubernetes v1.24)
      </li>
  </ul>
</div>
```

Reply:
@sftim I updated the PR to implement something very close to what you proposed. PTAL and let me know if it's okay

Reply:
![image](https://github.com/kubernetes/website/assets/18719127/4e628f19-219d-4f31-ac96-f4a0322f7d50)